gcal.py

```diff
@@ -23,6 +33,12 @@ def getEvents(text_message, group_id, start_date=None, days=None):
         result = json.loads(str(r.content, 'utf-8'))
         if (result.get('code') != 'SUCCESS'):
             raise result.get('message')
+        if year > 0:
+            filtered_result = []
+            for res in result.get('result'):
+                if res['name'].find(str(year) + ']') != -1:
```

Comment:
This is optional:
I think it will be better if you replace this `if` statement with a function like `if event_name_is_assignment_for_year(year)` or something. It will improve readibility of the code.

Reply:
Done